coffee_core/src/nurse/strategy.rs

```diff
@@ -74,3 +74,32 @@ impl Handler for GitRepositoryLocallyAbsentStrategy {
         }
     }
 }
+
+/// Stategy for migration of the repository global directory
+/// to a local directory for each network. See [related issue]
+///
+/// This is a strategy that returns the list of networks that
+/// need to be migrated to to the new repository configuration.
+///
+/// [issue]: https://github.com/coffee-tools/coffee/issues/234
```

Comment:
I think I mistyped it in my first suggestion 😃
```suggestion
/// [related issue]: https://github.com/coffee-tools/coffee/issues/234
```